src/main/containers/TestPanel.jsx

```diff
@@ -0,0 +1,13 @@
+import React, { useState } from 'react';
+
+const TestPanel = (props) => {
+  const [mockups, setData] = useState([props.data]);
```

Comment:
Any particular reason that we're passing along something that's a prop to our state?  I don't *think* it's a problem but the architecture feels weird.  I say either make TestPanel stateful and don't pass it along as a prop, or store it only in Panels and pass it as a prop.

Reply:
All of this will probably change as our state management needs become more complex and we start playing around more with the Context API. 

But the basic reasoning here is to allow for data to get passed _down_ to individual panels from the `Panels` container, and for each panel to maintain its own “version” of that data in state so we’re not mutating the original.

Reply:
Sounds good.  Point of PRs is to make our code better, I'm just trying to help out!  Do me the same favor.